cluster/test-deploy/ibmcloud/main.yaml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
I am not sure if this is needed at all. Are there any variables that the installer should use here?

Reply:
This might be a better place to inject some default values, such as `RESOURCE_GROUP`, etc.
I'm not sure what the best practice would be, default variables here, or in the `ipi-config-ibmcloud-commands.sh` script, which might be easier to pull from profile secret than in YAML.

I can remove if it's not necessary.

Reply:
For example, if we managed variables in `ipi-conf-ibmcloud-commands.sh` instead, we could remove this
```
--- a/ci-operator/step-registry/ipi/conf/ibmcloud/ipi-conf-ibmcloud-commands.sh
+++ b/ci-operator/step-registry/ipi/conf/ibmcloud/ipi-conf-ibmcloud-commands.sh
@@ -5,6 +5,8 @@ set -o errexit
 set -o pipefail
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
+RESOURCE_GROUP="${RESOURCE_GROUP:-Openshift-IPI-CI}"
+CIS_INSTANCE_CRN="${CIS_INSTANCE_CRN:-$(< ${CLUSTER_PROFILE_DIR}/cis-crn)}"
```

Reply:
well, those files will be mounted (if we implement it) just like the secrets. I guess your scripts have to source them, but either way, let's don't maintain them and just put the values in the script

Reply:
Okay, I will simply inject the value in the script instead

Reply:
Removed